python/vTestInterface.py

```diff
@@ -174,31 +181,35 @@ def getTestDataVCAST(enviroPath):
 
     # Now do normal tests
     for unit in api.Unit.all():
-        unitNode = dict()
-        unitNode["name"] = unit.name
-        try:
-            unitNode["path"] = unit.path
-        except:
-            pass
-        unitNode["functions"] = list()
-        for function in unit.functions:
-            functionNode = dict()
-            # Seems like a vcast dataAPI bug with manager.cpp
-            if function.vcast_name != "<<INIT>>":
-                # Note: the vcast_name has the parameterization only when there is an overload
-                functionNode["name"] = function.vcast_name
-                functionNode["tests"] = list()
-                for test in function.testcases:
-                    if test.is_csv_map:
-                        pass
-                    else:
-                        testInfo = generateTestInfo(test)
-                        functionNode["tests"].append(testInfo)
-
-                unitNode["functions"].append(functionNode)
-
-        if len(unitNode["functions"]) > 0:
-            testList.append(unitNode)
+        # we used to add these and throw them away in the typescript, nowe we don't add them
+        if unit.name != "uut_prototype_stubs":
+            unitNode = dict()
+            unitNode["name"] = unit.name
+            try:
+                unitNode["path"] = unit.path
+            except:
+                pass
+            unitNode["functions"] = list()
+            for function in unit.functions:
+                functionNode = dict()
+                # Seems like a vcast dataAPI bug with manager.cpp
+                if function.vcast_name != "<<INIT>>" and not function.is_non_testable_stub:
+                    # Note: the vcast_name has the parameterization only when there is an overload
+                    functionNode["name"] = function.vcast_name
+                    functionNode["parameterizedName"] = function.long_name
+                    globalListOfTestableFunctions.append (function.long_name)
```

Comment:
Can we make sure it's definitely `function.long_name` and not `function.vcast_name` that we want here? It's probably correct, as I got the correct behavior with this branch